### DISCUSSION TITLE
第一届跳舞大赛

### DISCUSSION THREAD

**Discussion by 1576655988** [share121/inter-knot#2235] [其他]
![emotion_download_1722874753104](https://github.com/user-attachments/assets/5811b93d-70ee-412d-aeb6-8eaa6286e720)

**Comment by 1576655988** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/496b41d101eec4c792e82da91fc29daf_2136519103324822738.png" alt="时光交响之章表情包17" width="75" >

**Comment by BlukxDD** [comment]
![H2442V7}3YR09HMX{UC91FG_tmb](https://github.com/user-attachments/assets/7700dccc-9da7-47f4-a703-b88b17e1b3a3)
![H2442V7}3YR09HMX{UC91FG_tmb](https://github.com/user-attachments/assets/f4a70245-e52b-4d28-8d8e-56ade8c927c2)
#

**Comment by KUYUE404** [comment]
![Uploading 1723712361011.gif…]()

**Comment by lllliufeng** [comment]
![MOGZXRSRC9WH%BBMF50C5J](https://github.com/user-attachments/assets/0a8a4355-b8dc-4e35-a597-6a43a851a450)
